docs/OV_Runtime_UG/PropertiesAPI.md

```diff
@@ -0,0 +1,229 @@
+
+Or the number of threads that would be used for inference in `CPU` device:
+
+@snippet snippets/ov_properties_api_5.cpp part5
+
+#### set_property()
+
+The only device that supports this method is [Multi-Device](supported_plugins/MULTI.md).
+
+## OpenVINO™ Properties API (Python)
+
+@sphinxdirective
+.. raw:: html
+
+    <div id="switcher-python" class="switcher-anchor">Python</div>
+@endsphinxdirective
+
+This section provides a high-level description of the process of querying of different device properties and configuration values. Refer to the [Hello Properties Device Python Sample](../../samples/python/hello_query_device/README.md) sources and the [Multi-Device Plugin documentation](supported_plugins/MULTI.md) for examples of using the OpenVINO™ Properties API in user applications.
+
+### Using the OpenVINO™ Properties API in Your Code
+
+The OpenVINO™ [Core](api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino-inference-engine-iecore) class provides the following API to query device information, set or get different device configuration properties:
+
+* [ie_api.IECore.available_devices](api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.available_devices) - Provides a list of available devices. If there are more than one instance of a specific device, the devices are enumerated with .suffix where suffix is a unique string identifier. The device name can be passed to all methods of the IECore class that work with devices, for example [ie_api.IECore.load_network](api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.load_network).
+* [ie_api.ieCore.get_property](api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.get_property) - Provides information about specific device.
+* [ie_api.IECore.get_config](api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.get_config) - Gets the current value of a specific configuration key.
+* [ie_api.IECore.set_config](api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.set_config)  - Sets a new value for the configuration key.
+
+The [ie_api.CompiledModel](api/ie_python_api/_autosummary/openvino.inference_engine.CompiledModel.html) class is also extended to support the Properties API:
+* [ie_api.CompiledModel.get_property](api/ie_python_api/_autosummary/openvino.inference_engine.CompiledModel.html#openvino.inference_engine.CompiledModel.get_property)
+* [ie_api.CompiledModel.get_config](latest/api/ie_python_api/_autosummary/openvino.inference_engine.CompiledModel.html#openvino.inference_engine.CompiledModel.get_config)
+* There is no method to call for set_config, but the equivalent action is described below.
+
+### Properties API in the IECore Class
+
+#### Get Available Devices
+
+```python
+from openvino.inference_engine import IECore
```

Comment:
We need to update python snippets.
Also I propose to use the same approach as for C++ code, and use snippets instead of hardcoded code.

Reply:
Certainly, we need to update python samples but I am not familiar with PythonAPI.

Reply:
It is not about samples. It is more about snippets.
Can we just add new snippets as was done here: https://github.com/openvinotoolkit/openvino/pull/10187/files#diff-6dba10fc49d7faf40671ce93fe8b59b680ce52e979ba8e3dc1761f64a1ef520eR1
I think you can ask @jiwaszki or @akuporos about any Python API questions.

Reply:
Created separate task 79843

Reply:
Why cannot we merge it in this PR?

Reply:
I am not familiar with Python API, module, variable or funcion namings and so on.